### ISSUE TITLE
Feature: Add SCSS support

### ISSUE THREAD

**Issue by susnux** [juliushaertl/theming_customcss#34]
It would be really great if SCSS would be supported, this would make the custom style scripts much cleaner.
E.g. `ScssPhp\ScssPhp\Compiler` could be used, at least this is how the *theming* app does it.

**Comment by juliushaertl** [comment]
I don't have plans on implementing that but contributions are very welcome.

Note that since server might drop scssphp as server side scss is no longer used in server (though the theming app might still) it would be good in that case to add it as a direct dependency to this app using something like https://github.com/coenjacobs/mozart to isolate it.

**Comment by kosmobent** [comment]
I've created a working version providing SCSS support using sass.js as compiler, see PR #62

**Comment by susnux** [comment]
Just as a update for my original feature request:

I do not think this is needed anymore. Modern CSS allows nesting without any precompiler.
All features that you previously needed SCSS for are now also available in plain vanilla CSS allowing for more standard solution without the need of a pre-compiler to often suffered from security vulnerabilities.

**Comment by kosmobent** [comment]
Yes, the primary use case of SCSS (nesting) is now also available in standard CSS. Nevertheless, there are still reasons why SCSS could be used instead of CSS, like mixins, flow control (@each, @for, ...) and more.

But regardless of the use cases of SCSS and whether they are required or not: Is there any drawback to adding SCSS support? I think it's still a nice feature for those who simply want to use SCSS instead of CSS, even if this number of people is small.
